images/capi/packer/config/windows/kubernetes.json

```
@@ -0,0 +1,5 @@
+{
+  "kubernetes_semver": "v1.19.1",
```

```suggestion
  "kubernetes_semver": "v1.19.2",
```
now? Things move fast 😅